app/Http/Controllers/Api/V1/OrganisationController.php

```diff
@@ -1258,6 +1258,136 @@ public function inviteUser(OrganisationInviteUser $request, int $id): JsonRespon
+     *          ),
+     *      ),
+     *      @OA\Response(
+     *          response=201,
+     *          description="Success",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string", example="success"),
+     *              @OA\Property(property="data", type="integer", example="1"),
+     *          ),
+     *      ),
+     *      @OA\Response(
+     *          response=400,
+     *          description="Invalid argument(s)",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string", example="Invalid argument(s)")
+     *          )
+     *      ),
+     *      @OA\Response(
+     *          response=500,
+     *          description="Error",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string", example="error")
+     *          )
+     *      )
+     * )
+     */
+    public function custodianInviteUser(OrganisationInviteUser $request, int $id): JsonResponse
+    {
+        try {
+            $org = Organisation::findOrFail($id);
+            if (!$org->system_approved) {
+                return $this->ForbiddenResponse('The organization was not approved.');
+            }
+
+            $input = $request->all();
+            if (User::where("email", $input['email'])->exists()) {
+                return $this->ConflictResponse();
+            }
+            
+            $loggedInUserId = $request->user()->id;
+            $loggedInUser = User::where('id', $loggedInUserId)->first();
+            if ($loggedInUser->user_group !== User::GROUP_CUSTODIANS) {
+                return $this->ForbiddenResponse('Non-custodian users cannot invite members via this endpoint.');
+            }
+
+            $unclaimedUser = RMC::createUnclaimedUser([
+                'firstname' => $input['first_name'],
+                'lastname' => $input['last_name'],
+                'email' => $input['email'],
+                'organisation_id' => (isset($input['user_group']) && $input['user_group'] === User::GROUP_ORGANISATIONS) ? $id : 0,
+                'is_delegate' => isset($input['is_delegate']) ? $input['is_delegate'] : 0,
+                'user_group' => isset($input['user_group']) ? $input['user_group'] : 'USERS',
+                'role' => isset($input['role']) ? $input['role'] : null,
+            ]);
+
+            if (isset($input['department_id']) && $input['department_id'] !== 0 && $input['department_id'] != null) {
+                UserHasDepartments::create([
+                    'user_id' => $unclaimedUser->id,
+                    'department_id' => $request['department_id'],
+                ]);
+            };
+
+            $email = [
+                'type' => 'USER',
+                'to' => $unclaimedUser->id,
+                'by' => $id,
+                'identifier' => 'custodian_user_invite',
+                'custodianId' => $loggedInUserId,
+            ];
+
+            Affiliation::create([
```

Comment:
This needs a check to ensure we're not creating affiliations for org users too. Org users don't have a registry, therefore this will blow up if it's tried.

Reply:
Upon creation, a new record is also generated in the registers that link to users and affiliations.